File: 2__C++_Program/5__conversion_length.cpp
#include <iostream>

using namespace std;

int main() {
    // Declare variables and input cm
    int cm, m, leftover_cm;
    cout << "Enter length in centimeter: "; cin >> cm;
    
    // Calculate cm to m
    m = cm / 100;
    leftover_cm = cm % 100;
    cout << "Cm to m: " << m << "m" << " and " << leftover_cm << "cm" << endl;
    return 0;
}

File: 5__JavaScript_Program/5__conversion_length.js
const prompt = require('./node_modules/prompt-sync')({ sigint: true });

// Input cm and print m
const cm = Number(prompt("Enter length in centimeter: "));

const m = Math.floor(cm / 100);
const leftover_cm = cm % 100;
console.log(`Cm to m: ${m}m and ${leftover_cm}cm`);